bin/lib/zopen-build

```diff
@@ -11,6 +11,29 @@
 # Functions section
 #
 
+#
+# asciiecho: we are in the process of starting to use 'echo' from coreutils.
+# This echo will have a slightly different behaviour than the standard echo in that
+# the file will now be tagged as ascii instead of ebcdic
+# For compatibility, check the tag of the file after the echo and iconv the file IFF it is IBM-1047
+#
+asciiecho()
```

Comment:
A question, is it required at places where it is writing to .env file as well? for instance:
echo "export PATH=\"\${PWD}/bin:\$PATH\"" >> "${ZOPEN_INSTALL_DIR}/.env"

I see that .env file is in IBM-1047, in install dirs

Reply:
Hmm, I'm thinking we probably should keep the .env files as ibm-1047 in case the user forgets to set _BPXK_AUTOCVT=ON

Reply:
hm. i expect we might have dependencies on _BPXK_AUTOCVT=ON elsewhere... should we alternately put a check into the script to verify _BPXK_AUTOCVT (and perhaps any other env vars we want) are already set? It would also mean we don't have to replicate that in all the .env files we generate... 
Does anyone run their shell without _BPXK_AUTOCVT=ON from this group?

Reply:
can we have this as a separate issue? I do like the idea of having a completely 'clean' ASCII set of trees (other than executables) that we can validate against to catch any problems in our tools?